RELEASE_NOTES.md

```diff
@@ -1,42 +1,279 @@
+
+### Breaking Changes
+
+#### AIX Virtualization Improvements
+
+The Ohai :Virtualization plugin on AIX systems will now properly return the `lpar_no` and `wpar_no` values as Integers instead of Strings. This makes the data much easier to work with in cookbooks, but may be a breaking change depending on how AIX users consumed these values.
+
+#### 32bit RHEL/CentOS 6 Support
+
+Chef Infra Client 17 packages will no longer be produced for 32bit RHEL/CentOS 6 systems. RHEL/CentOS 6 became EOL November 2020. We are extending support for 64 bit RHEL/CentOS 6 until Chef Infra Client 18 (April 2022) or when upstream platform or library changes prohibit us from building on these EOL systems.
+
+#### Chef Client As A Service on Windows
+
+Based on customer feedback and observations in the field we've removed the ability to run the Chef Infra Client as a service on Windows nodes. We've seen the service manager for the Chef Infra Client consume excessive memory, hang preventing runs, or prevent nodes from updating to new client releases properly. We've always seen significantly better reliability by running Chef Infra Client as a scheduled task on Windows and in July of 2006 we introduced warnings to the [chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) when running as a service. The ability to setup the client as a service was later removed from the cookbook entirely in October of 2017.
+
+For customers current running Chef Infra Client as a service we suggest migrating to scheduled task based execution. This allows for complex scheduling scenarios not possible with simple services such, as the ability to skip Chef Infra Client execution if a system is on battery or to run the Chef Infra Client as soon as a system boots to ensure configuration.
+
+Chef Infra Client can be configured to run as a scheduled task using the [chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) or ideally using the [chef_client_scheduled_task resource](https://docs.chef.io/resources/chef_client_scheduled_task/) built into Chef Infra Client 16 or later. For users already running as a service setting up the scheduled task and then stopping the existing service can be performed within a Chef Infra Client run to migrate systems.
```

Comment:
If using the chef_client_scheduled_task resource is the preferred solution, then we should list it first.

Reply:
The reason for mentioning the chef-client cookbook is just to show that this is not a new warning.